database/mongodb.go

```diff
@@ -240,3 +240,14 @@ func (m *MongoDB) GetMessages() ([]model.AdminMessage, error) {
 	err := m.messages.Find(bson.M{}).All(&messages)
 	return messages, err
 }
+
+// UserExists tests if a given user exists in the admin database
+func (m *MongoDB) UserExists(uname string) bool {
+	user := model.User{}
+	err := m.users.Find(bson.M{"name": uname}).One(&user)
+	_ = user
```

Comment:
Also, to avoid needing to do this, I think we can use mgo's `Count` method instead of `One`.